src/main/java/io/logz/jmx2graphite/Jmx2GraphiteJavaAgent.java

```diff
@@ -38,7 +38,7 @@ public static void premain(String agentArgument, Instrumentation instrumentation
 
         Jmx2GraphiteConfiguration jmx2GraphiteConfiguration = new Jmx2GraphiteConfiguration(finalConfig);
 
-        Jmx2Graphite main = new Jmx2Graphite(jmx2GraphiteConfiguration);
+        Jmx2Graphite main = new Jmx2Graphite(jmx2GraphiteConfiguration, true);
```

Comment:
Can you please explain this choice?

Reply:
yes, see above, but I will remove it from this PR, thanks.